src/components/Exercise.tsx

```diff
@@ -94,34 +102,35 @@ export const Exercise = ({
       <ExerciseIcons exercise={exercise} topicUrl={props.topicUrl} errataUrl={props.errataUrl} /> : null}
     showTotalQuestions={legacyStepRender}
   >
-    <Preamble exercise={exercise} />
+    <div ref={container}>
```

Comment:
This wrapper doesn't feel ideal but I ran into forwardedRef issues that I didn't quite have time to get working. Would like to do a follow up, seems possible to attach it to StyledTaskStepCard instead.

Reply:
yea i remember styled-components doesn't play nice with forwardRef. i've done it before but i'm not sure if we're on the same version still. what you have is fine for now